examples/react/base/src/routes.tsx

```diff
@@ -0,0 +1,29 @@
+import { lazy, PropsWithChildren } from "react";
+import { Switch, Route } from "wouter";
+import { Layout } from "@/ui/pages/layout";
+
+const Home = lazy(() => import("@/ui/pages"));
+const About = lazy(() => import("@/ui/pages/about"));
+const NotFound = lazy(() => import("@/ui/pages/not-found"));
+
+function RouteWithLayout({ children }: PropsWithChildren<{ path: string }>) {
+  return <Layout>{children}</Layout>;
+}
+
```

Comment:
Is it too much if we have RouteWithLayout.tsx?

I mean, I thought we could do like below.

src
|__routes
|____index.tsx
|____RouteWithLayout(Or RouteWrapper)
|______index.tsx